hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -389,291 +377,6 @@ func (mh *Memcached) implementingWebhooks() {
 	pkg.CheckError("adding imports", err)
 }
 
-// implementingController will customize the Controller
-func (mh *Memcached) implementingController() {
-	controllerPath := filepath.Join(mh.ctx.Dir, "controllers", fmt.Sprintf("%s_controller.go",
-		strings.ToLower(mh.ctx.Kind)))
-
-	// Add imports
-	err := kbutil.InsertCode(controllerPath,
-		"import (",
-		importsFragment)
-	pkg.CheckError("adding imports", err)
-
-	// Add RBAC permissions on top of reconcile
-	err = kbutil.InsertCode(controllerPath,
-		"/finalizers,verbs=update",
-		rbacFragment)
-	pkg.CheckError("adding rbac", err)
-
-	// Replace reconcile content
-	err = kbutil.ReplaceInFile(controllerPath,
-		`"sigs.k8s.io/controller-runtime/pkg/log"`,
-		`ctrllog "sigs.k8s.io/controller-runtime/pkg/log"`,
-	)
-	pkg.CheckError("replacing controller log import", err)
-	err = kbutil.ReplaceInFile(controllerPath,
-		"_ = log.FromContext(ctx)",
-		"log := ctrllog.FromContext(ctx)",
-	)
-	pkg.CheckError("replacing controller logger construction", err)
-
-	// Add reconcile implementation
-	err = kbutil.ReplaceInFile(controllerPath,
-		"// TODO(user): your logic here", reconcileFragment)
-	pkg.CheckError("replacing reconcile content", err)
-
-	// Add helpers funcs to the controller
-	err = kbutil.InsertCode(controllerPath,
-		"return ctrl.Result{}, nil\n}", controllerFuncsFragment)
-	pkg.CheckError("adding helper methods in the controller", err)
-
-	// Add watch for the Kind
-	err = kbutil.ReplaceInFile(controllerPath,
-		fmt.Sprintf(watchOriginalFragment, mh.ctx.Group, mh.ctx.Version, mh.ctx.Kind),
-		fmt.Sprintf(watchCustomizedFragment, mh.ctx.Group, mh.ctx.Version, mh.ctx.Kind))
-	pkg.CheckError("replacing add controller to manager", err)
-}
-
-// nolint:gosec
-// implementingAPI will customize the API
-func (mh *Memcached) implementingAPI() {
-	err := kbutil.InsertCode(
-		filepath.Join(mh.ctx.Dir, "api", mh.ctx.Version, fmt.Sprintf("%s_types.go", strings.ToLower(mh.ctx.Kind))),
-		fmt.Sprintf("type %sSpec struct {\n\t// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster\n\t// Important: Run \"make\" to regenerate code after modifying this file", mh.ctx.Kind),
-		`
-
-	// Size defines the number of Memcached instances
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
```

Comment:
We need add on top of the scaffold done now the marker ` +operator-sdk:csv:customresourcedefinitions:type=spec` it is specific for SDK / bundle